src/app/packages/core/utils/ansyn-logger.ts

```diff
@@ -0,0 +1,68 @@
+export type Severity = 'CRITICAL' | 'ERROR' | 'WARNING' | 'INFO' | 'DEBUG'
+
+interface MinimalLogger{
+	error(msg: string);
+	warn(msg: string);
+	info(msg: string);
+}
+
+export class AnsynLogger implements MinimalLogger {
+	protected static _activeLogger = new AnsynLogger('ENV');	// default logger (unknown environment)
+	// when extending AnsynLogger - override the following get/set functions
+	public static set activeLogger(logger: AnsynLogger) {
+		AnsynLogger._activeLogger = logger;
+	}
+	public  static get activeLogger(): AnsynLogger {
+		return AnsynLogger._activeLogger;
+	}
+
+	env: string;
+	standardPrefix: string;
+
+
+
+	constructor(env: string) {
+		this.standardPrefix = `Ansyn[${this.env}]`
+	}
+
+	critical(msg: string) {
```

Comment:
you never call `critical`, `error`, `warn`, `info`, and `debug`, only the methods of the static `activeLogger`

Keep the static stuff, they are fine, but move the logging into  new class that implements `MinimalLogger`. Too much redundancy here

Reply:
FAD, see #947